src/notebook/cells/editor.ts

```diff
@@ -416,7 +415,7 @@ class CodeCellEditorWidget extends CodeEditorWidget implements ICellEditorWidget
       model.source = newValue;
     }
     this.textChanged.emit({
-      line, ch, chHeight, chWidth, coords, position, oldValue, newValue
+      line, ch, chHeight, chWidth, coords, position, newValue
```

Comment:
We weren't using the `oldValue` anywhere.  I agree that the expense is not worth adding it.